CHANGELOG.md

```diff
@@ -1,24 +1,40 @@
 # Changelog
 Changelog for NeoFS Node
 
-## [Unreleased]
+## [Unreleased] - Anmado (안마도, 鞍馬島)
+
+### Added
+
+### Changed 
+
+### Fixed
+
+### Removed
+
+### Updated
+
+### Updating from v0.32.0
+
+## [0.32.0] - 2022-09-14 - Pungdo (풍도, 楓島)
 
 ### Added
 - Objects counter metric (#1712)
 - `meta` subcommand to `neofs-lens` (#1714)
 
 ### Changed
 - `neofs-cli object put`'s object ID output has changed from "ID" to "OID" (#1296)
-- `neofs-cli container set-eacl` command now pre-checks container ACL's extensibility (#1652) 
+- `neofs-cli container set-eacl` command now pre-checks container ACL's extensibility (#1652)
 
 ### Fixed
 - Panic on write-cache's `Delete` operation (#1664)
```

Comment:
seems like `0.31.0` -> `0.32.0` changelog is not that short: at least new metrics (logical), tree ACL service changes, blobocnicza fix, etc.

Reply:
Sad that these changes aren't in `Unreleased`. We have to watch this more closely.

Reply:
Added mentioned changes. Maybe smth else @carpawell @fyrchik?

Reply:
Added
- Removal of trees on container removal (#1630)
- Logging new epoch events (#1763)
- Timeout for streaming operations (#1746)
- `neofs-adm` is able to dump hashed from a custom zone (#1748)
- Empty filename support in the Tree Service (#1698)
- Flag to `neofs-cli` list commands that allows printing attributes (#1649)

Changed
- Tree service does not restrict depth in `GetSubTree` (#1753)
- `neofs-adm` registers contract hashes in hex and address string formats (#1749)
- Container list cache (#1632)
- Blobstore components are unified (#1584, #1686, #1523)

FIxed
- object expiration (#1670)
- placement policy parser (#1775)
- Tree service timeout logs (#1759)
- Write-cache's object flushing (#1745)
- Active blobovniczas caching (#1691)
- `neofs-adm` TX waiting (#1738)
- `neofs-adm` registers contracts with a minimal GAS payment (#1683)
- Permissions of the file created by `neofs-cli` (#1719)
- `neofs-adm` creates TX with a high priority attribute (#1702)
- Restart after a hard reboot (#1647)

the wording needs to be double checked. also, i would merge the records about the metrics since they are about the same issue. also, i would describe config changes after the blobstor refactor

Reply:
1. Fix config parsing (blobovnicza type was invalid)
2. GetSubTree now works for arbitrary tree depth
3. neofs-adm can dump hashes in arbitrary zone
4. blobstor configuration refactoring, described in docs

Reply:
Thanks, added.